src/baseCommand.ts

```diff
@@ -67,11 +71,11 @@ export abstract class BaseCommand extends Command {
 
         networkPreset.network.name = flags.network;
 
-        this.api = new UNSCLIAPI(networkPreset);
+        this.api = new UNSCLIAPI(networkPreset, flags.node);
 
         this.client = new Client(networkPreset);
 
-        this.unsClient = new UNSClient(networkName);
+        this.unsClient = new UNSClient(networkName, flags.node);
```

Comment:
you must use `Network` enum as first parameter of constructor. 
It's currently possible due to `strict mode` disabled.

Reply:
flags.node is an url, not a network name

Reply:
> as first parameter of constructor.